wpeverest/user-registration
# Enhance- Re-initialize form after full page load

### All Submissions:






### Changes proposed in this Pull Request:
Previously our registration form was not working in third-party popup plugins which load popup through ajax and it was mandatory to load our registration form scripts after the third party popup plugin's popup ajax was fired in order for this issue to be solved. This PR fixes this issue.
Closes #196  .

### How to test the changes in this Pull Request:

1. Use our registration form shortcode inside third party popup plugins like elementor.
2. Vist the popup on the frontend and try to submit the form.
3. Verify if the errors in the form are shown correctly and the form is submitted after clicking the submit button.

### Types of changes:




### Other information:




### Changelog entry

> Enhance- Re-initialize form after full page load.
